x/wasm/keeper/query_plugins_test.go

```diff
@@ -445,6 +453,40 @@ func TestBankQuerierAllMetadata(t *testing.T) {
 	assert.Equal(t, exp, got)
 }
 
+func TestBankQuerierAllMetadataPagination(t *testing.T) {
+	var requestedPagination *query.PageRequest
+	mock := bankKeeperMock{GetDenomsMetadataFn: func(ctx context.Context, req *banktypes.QueryDenomsMetadataRequest) (*banktypes.QueryDenomsMetadataResponse, error) {
+		// t.Log("test")
+		// t.Log(req == nil)
+		requestedPagination = req.Pagination
+		return &banktypes.QueryDenomsMetadataResponse{
+			Metadatas: []banktypes.Metadata{},
+			Pagination: &query.PageResponse{
+				NextKey: nil,
+			},
+		}, nil
+	}}
+
+	ctx := sdk.Context{}
+	q := keeper.BankQuerier(mock)
+	pagination := &wasmvmtypes.PageRequest{
```

Comment:
nit: please remove the object and just use the expected values in the assert lines below:
`assert.Equal(t, uint64(10), requestedPagination.Limit)`

or use `&query.PageRequest{..}` and move the declaration down as described below